kubernetes/linux/main.sh

```diff
@@ -417,12 +424,21 @@ if [[ ( ( ! -e "/etc/config/kube.conf" ) && ( "${CONTAINER_TYPE}" == "Prometheus
 
       if [ -e "integration_osm_config_env_var" ]; then
             cat integration_osm_config_env_var | while read line; do
-                  echo $line >> ~/.bashrc
+                  echo $line >> /opt/env_vars
```

Comment:
From what I see, /opt/env_vars is not sourced after writing to it. Will this work?

Reply:
Yeah, it works because right after the loop `source integration_osm_config_env_var` is called (like the other loops around it). This should be slightly more efficient than `source /opt/env_vars` because integration_osm_config_env_var will be a smaller file

Reply:
So then why do we have to write it to /opt/env_vars?

Reply:
So any other processes in the container will also get the change (liveness probe and when user execs into a container). (I guess for the same reason environment variables were previously written to ~/.bashrc)

Reply:
But we are not sourcing after writing to this file (/opt/env_vars)